static/app/views/onboarding/targetedOnboarding/types.ts

```diff
@@ -27,3 +24,23 @@ export type StepDescriptor = {
   title: string;
   hasFooter?: boolean;
 };
+
+export type ClientState = {
+  // map from platform id to project id. Contains projects ever created by onboarding.
+  platforms: {[key in PlatformKey]?: string};
```

Comment:
@Neo-Zhixing I wonder if we could do a name like `platformToProjectIdMap` that's more descriptive